### DISCUSSION TITLE
Using d3.geoMercator, i got NaN

### DISCUSSION THREAD

**Discussion by chemistwang** [d3/d3#3819] [Q&A] [answered]
Here is my function:   `const projectionFn = d3.geoMercator().center(pjCenter).scale(pjScale).translate([0, 0]);`. But i got NaN, i was wondering the reason why.

**Comment by Fil** [comment]
what are the values of pjCenter and pjScale?

**Reply by chemistwang** [reply]
pjCenter is `[117.226808, 31.849411]`, pjScale is `600`

**Comment by Fil** [comment, marked as answer]
This works for me; please share most context as to how you “got NaN”.
![Capture d’écran 2024-01-30 à 09 46 33](https://github.com/d3/d3/assets/7001/672769f2-1c64-47e2-8b4c-a109fe15d33f)

**Reply by chemistwang** [reply]
I rechecked it again and i found this is my fault. The projectionFn parameter requires a one-dimensional array, but I used a single number. Thank you!
